examples/comfyui/README.md

```diff
@@ -0,0 +1,71 @@
+# finegrain-comfyui
+
+Finegrain API ComfyUI custom nodes
+
+## Installation
+
+### Requirements
+
+1. Ensure you have Python 3.10 or later installed.
```

Comment:
But the Finegrain client requires 3.12+ https://github.com/finegrain-ai/finegrain-python/blob/a03cff35d17d5351cb7179cfa71c2e2eb5e7b1ca/finegrain/pyproject.toml#L13